HSLdevcom/jore-map-ui
# searching other line + fixed search list and routePath list

Resolves: https://github.com/HSLdevcom/jore-map-ui/issues/88

Testing:

- check scrollbar at index page

- search for line 64 and check scrollbar etc.
